pkg/test/gcp/gcp.go

```diff
@@ -52,15 +52,17 @@ func (v *EnvVar) Set(s string) {
 }
 
 var (
+	TestFolderID                        = EnvVar{Key: "TEST_FOLDER_ID"}
+	TestFolder2ID                       = EnvVar{Key: "TEST_FOLDER_2_ID"}
 	TestOrgID                           = EnvVar{Key: "TEST_ORG_ID"}
 	TestBillingAccountID                = EnvVar{Key: "TEST_BILLING_ACCOUNT_ID"}
 	IAMIntegrationTestsOrganizationID   = EnvVar{Key: "IAM_INTEGRATION_TESTS_ORGANIZATION_ID"}
 	IAMIntegrationTestsBillingAccountID = EnvVar{Key: "IAM_INTEGRATION_TESTS_BILLING_ACCOUNT_ID"}
+	TestAttachedClusterName             = EnvVar{Key: "TEST_ATTACHED_CLUSTER_NAME"}
```

Comment:
I think I forgot to delete this variable. This is no longer used anywhere.

Reply:
Thanks - removed!

Reply:
Actually, looks like we still use ATTACHED_CLUSTER_NAME in the samples (which we also run / can run in the unified tests)

Reply:
Samples test for this resource is disabled: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/config/tests/samples/create/samples_test.go#L69-L73.